iwyu_tool_test.py

```diff
@@ -19,6 +19,23 @@
 except ImportError:
     from io import StringIO
 
+class MockProcess(object):
+    def __init__(self, block, content):
+        self.block = block
+        self.content = content
+    def poll(self):
+        if self.block > 0:
+            from time import sleep
```

Comment:
Imports at the top

Reply:
Also, this implementation seems weird -- `poll` should never block. Maybe it would be more "true" to keep track of time and return None until `self.block` has elapsed?